### PR TITLE
Added yaml files for offline migration with public hdd image

### PR DESCRIPTION
**Pull request by tinawang123**
Added yaml files for offline migration with public hdd image

- Related ticket: https://progress.opensuse.org/issues/150974
- Verification run: http://openqa.suse.de/t12873898
http://openqa.suse.de/t12873901
http://openqa.suse.de/t12873902
http://openqa.suse.de/t12873904
- MR: https://gitlab.suse.de/qe-yam/openqa-job-groups-migrations/-/merge_requests/393

### CODE REVIEW

File: schedule/migration/migration_offline_scc_deregistration_ppc64le.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - installation/first_boot
+  - migration/post_upgrade
+  - console/system_prepare
+  - console/consoletest_setup
+  - console/zypper_lr
+  - console/check_os_release
+  - console/check_system_info
+  - console/hostname
+  - console/force_scheduled_tasks
+  - console/scc_deregistration
```
Comment (lemon-suse) [inline schedule/migration/migration_offline_scc_deregistration_ppc64le.yaml:38]: I think we can add condition to control these modules loaded or not for publish qcow or not, so this yaml file can be used for two phase of continuous migration test.

Comment (tinawang123) [reply schedule/migration/migration_offline_scc_deregistration_ppc64le.yaml:38]: As we discussed. The second part uses online migration. They cannot share the same yaml file.

**Review by chcao — APPROVED** [review]
LGTM. It seems you haven't submit the related MR, we can not access it.

File: schedule/migration/migration_offline_scc_deregistration_ppc64le.yaml (hunk 1 above)
Comment (tinawang123) [reply schedule/migration/migration_offline_scc_deregistration_ppc64le.yaml]: Updated

File: schedule/migration/migration_offline_scc_deregistration_aarch64.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  - console/force_scheduled_tasks
+  - '{{scc_deregistration}}'
+  - shutdown/grub_set_bootargs
+  - shutdown/cleanup_before_shutdown
+  - shutdown/shutdown
+conditional_schedule:
+  scc_deregistration:
+    SCC_DEREGISTER:
+      1:
+        - console/scc_deregistration
```
Comment (lemon-suse) [inline schedule/migration/migration_offline_scc_deregistration_aarch64.yaml:46]: The condition should include all modules for publish qocw:
  - console/scc_deregistration
  - shutdown/grub_set_bootargs
  - shutdown/cleanup_before_shutdown
  - shutdown/shutdown

Comment (tinawang123) [reply schedule/migration/migration_offline_scc_deregistration_aarch64.yaml:43]: Updated

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**